til-domain/src/main/java/com/til/application/interview/InterviewService.java

```diff
@@ -60,20 +63,20 @@ public InterviewCodeDto createInterview(InterviewCreateDto interviewCreateDto) {
 
     @Transactional
     public InterviewCodeDto createSpeechInterview(SpeechInterviewCreateDto speechInterviewCreateDto) {
-        checkProcessingInterviewByUserId(speechInterviewCreateDto.userId());
+        checkCreatingOrProcessingInterviewByUserId(speechInterviewCreateDto.userId());
```

Comment:
P4: 아래처럼 변경해봐도 좋을 것 같습니다!
```suggestion
        checkInterviewStatusByUserId(speechInterviewCreateDto.userId());
```

Reply:
해당 메소드는 `checkInterviewStatusByUserId` 로는 표현이 잘 안되는것 같아서 좀 더러워도 기존 이름 유지하겠습니다!